tiflash/use-tiflash.md

```diff
@@ -244,3 +244,44 @@ TiFlash does not support push-down calculations in the following situations:
 - If an aggregate function or a `WHERE` clause contains expressions that are not included in the list above, the aggregate or related predicate filtering cannot be pushed down.
 
 If a query encounters unsupported push-down calculations, TiDB needs to complete the remaining calculations, which might greatly affect the TiFlash acceleration effect.
+
+## Use the MPP mode
+
+TiFlash supports using the MPP mode to execute queries, which introduces cross-node data exchange (data shuffle process) into the computation. The MPP mode is turned on by default and can be turned off by setting the value of the global/session variable [`tidb_allow_mpp`](/system-variables.md#tidb_allow_mpp-new-in-v50-ga) to `0` or `OFF`.
+
+```shell
+set @@session.tidb_allow_mpp=0
+```
+
+MPP mode supports the following physical algorithms: Broadcast Hash Join, Shuffled Hash Join, and Shuffled Hash Aggregation. The optimizer automatically determines which algorithm to be used in a query. To check the specific query execution plan, you can execute the `EXPLAIN` statement.
+
+The following statement takes the table structure in the TPC-H test set as an example：
+
+```sql
+mysql> explain select count(*) from customer c join nation n on c.c_nationkey=n.n_nationkey;
```

Comment:
Suggest removing this. In docs-cn, we can remove it in a separate PR. This is only a minor suggestion.
```suggestion
explain select count(*) from customer c join nation n on c.c_nationkey=n.n_nationkey;
```